DEVELOP.rst

```diff
@@ -88,11 +88,17 @@ To compile run::
 
     ./gradlew compileJava
 
+Before running, a config file for elasticsearch's log4j logging is required. Create an empty default with::
+
+    touch config
+
+If you want to configure elasticsearch or its logging, see the `Elasticsearch reference`_ .
```

Comment:
pls dont refer to ES documentation, rather use: [https://crate.io/docs/en/stable/configuration.html#logging](https://crate.io/docs/en/stable/configuration.html#logging)